diff -u a/input.py b/input.py
--- a/input.py	2023-06-14 19:28:02.826432124 -0700
+++ b/input.py	2023-06-14 19:28:12.621431539 -0700
@@ -231,7 +231,7 @@
       # node-gyp produces ebcdic files, do not use that mode.
       build_file_contents = open(build_file_path, 'r').read()
     else:
-      build_file_contents = open(build_file_path, 'rU').read()
+      build_file_contents = open(build_file_path, 'r').read()
   else:
     raise GypError("%s not found (cwd: %s)" % (build_file_path, os.getcwd()))
 
